src/main/java/net/dv8tion/jda/webhook/WebhookClientBuilder.java

```diff
@@ -34,7 +35,7 @@
 public class WebhookClientBuilder
 {
     public static final OkHttpClient.Builder DEFAULT_HTTP_BUILDER = new OkHttpClient.Builder();
-    private static final Pattern WEBHOOK_PATTERN = Pattern.compile("webhooks\\/(\\d+)\\/([\\w-]+)");
```

Comment:
Don't you wanna go with `(?:www\\.)?` ? At least escape the dot. Otherwise you match any prefix.

Also: I'm assuming `/api(?:/\\d+)?/` is to allow api version? If so: api version is prepended by `v` so like `/api/v6/` (see docs here: <https://discordapp.com/developers/docs/reference>)

Also: what is the last part (after id/token) about?

Reply:
>Don't you wanna go with (?:www\\.)?

No, that would exclude canary, ptb and whatnot

>what is the last part (after id/token) about?

This allows matching of routes that end with say `/slack` or `/github`

Reply:
Oh right. other 2 points are still valid tho (escaping dot and adding `v` to version match)